### PR TITLE
Fix prepare release process

### PR DESCRIPTION
**Pull request by aanm**
cc @michi-covalent do we need to update the steps in https://github.com/cilium/release?

### CODE REVIEW

File: install/kubernetes/Makefile.values [hunk 1]
```diff
@@ -9,7 +9,7 @@ ifeq ($(RELEASE),yes)
     export PULL_POLICY:=IfNotPresent
     export CILIUM_REPO:=quay.io/cilium/cilium
     export CLUSTERMESH_APISERVER_REPO:=quay.io/cilium/clustermesh-apiserver
-    export HUBBLE_RELAY_REPO:=quay.io/cilium/hubble-rela
+    export HUBBLE_RELAY_REPO:=quay.io/cilium/hubble-relay
```
Comment (aanm) [inline install/kubernetes/Makefile.values:12]: @joestringer I noticed this after tagging v1.13.0-rc3. This is fixed in the helm chart.

**Review by michi-covalent — APPROVED** [review]

**Review by kaworu — APPROVED** [review]

File: contrib/release/start-release.sh [hunk 2]
```diff
@@ -113,12 +113,12 @@ main() {
...
       $DIR/prep-changelog.sh "$old_version" "$version"
     fi
-    git commit -a -s -m "Prepare for release $version"
 
     logecho "Next steps:"
-    logecho "* Check the new release commit with 'git show'"
+    logecho "* Check all changes and add to a new commit"
     logecho "  * If this is a prerelease, create a revert commit"
     logecho "* Push the PR to Github for review ('submit-release.sh')"
+    logecho "* Close https://github.com/cilium/cilium/projects/$old_proj"
```
Comment (joestringer) [inline contrib/release/start-release.sh]: This was already done by the cilium/release script before we even run this script right?

Comment (aanm) [reply contrib/release/start-release.sh]: I think yes, I'll remove this line

File: contrib/release/start-release.sh [hunk 3]
```diff
@@ -113,12 +113,12 @@ main() {
     else
       $DIR/prep-changelog.sh "$old_version" "$version"
     fi
-    git commit -a -s -m "Prepare for release $version"
```
Comment (joestringer) [inline contrib/release/start-release.sh:116]: I'll probably just fix this issue by making it commit the authors separately during the next RC prep. Do we need to revert this?

**Review by joestringer — COMMENTED** [review]
Commit 2, 3 makes sense but I was thinking I might just fix the issue that makes commit 1 necessary. I would like us to be moving more and more to an automated process like that rather than pushing more in the direction of requiring a human to be in the process at each step.

**Review by joestringer — APPROVED** [review]
We could alternatively just merge this and I'll separately include re-enabling the commit + another commit for authors etc. in a new PR after I do the next RC

**Comment by aanm** [comment]
> We could alternatively just merge this and I'll separately include re-enabling the commit + another commit for authors etc. in a new PR after I do the next RC

@joestringer Unfortunately the commit for the authors will always have to be reset because we might need to modify the authors file.

**Comment by joestringer** [comment]
@aanm :thinking: What I have in mind is we automatically create two commits - one for authors, then one for the rest. Will that work?

**Comment by aanm** [comment]
@joestringer whatever commit we do for authors, we might still have to amend it / fix it afterwards. Fixing the authors file is easier if the changes are not committed. Do you have other way to fix the authors file that I don't know about? :smile:

**Comment by joestringer** [comment]
Yeah, either don't fix it or inspect + create autosquash commits afterwards and rebase. Lately given that CNCF accepts aliases for code contributions, I have not been amending the authors file beyond the git commit data.

**Pull request merged**